english/bear/bear.md

```diff
@@ -9,3 +9,6 @@ When she slaps you in the face...
 [You vomit on top of her](../vomit/vomit.md)
 
 ..., holding your hand to your red cheek [you say] (../slap-response/slap-response.md)
+
+all is not what it seems in this story.
+you're talking about bears, but i'm talking about [bears](https://twitter.com/ChicagoBears?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.